lib/Mailer/Mailer.php

```diff
@@ -74,7 +74,7 @@ private function buildMailer() {
           $this->sender,
           $this->replyTo,
           new MailPoetMapper(),
-          new AuthorizedEmailsController($this->settings, new Bridge)
+          ContainerWrapper::getInstance()->get(AuthorizedEmailsController::class)
```

Comment:
Why didn't you inject it directly?

Reply:
There are a couple of places that use `new Mailer()`. I'd have to refactor all of those and it wouldn't be a small refactor.